#!/bin/sh /usr/share/dpatch/dpatch-run
## 02-libtool.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Fixes generation of shared libraries for plugins on !i386 architectures
## DP: as suggested by Kris Jurka <jurka@ejurka.com>
## DP: (Closes: #380115, #380318, #380321).

@DPATCH@

diff -Naur xmms-1.2.10+20060801.orig/ltmain.sh xmms-1.2.10+20060801/ltmain.sh
--- xmms-1.2.10+20060801.orig/ltmain.sh	2006-03-11 18:49:04.000000000 +0000
+++ xmms-1.2.10+20060801/ltmain.sh	2006-08-11 18:49:23.000000000 +0000
@@ -24,6 +24,9 @@
 # configuration script generated by Autoconf, you may include it under
 # the same distribution terms that you use for the rest of that program.
 
+EGREP="grep -E"
+max_cmd_len=32768
+
 basename="s,^.*/,,g"
 
 # Work around backward compatibility issue on IRIX 6.5. On IRIX 6.4+, sh
